Assets/VRTK/Scripts/Interactions/VRTK_ControllerActions.cs

```diff
@@ -122,6 +127,25 @@ public virtual void OnControllerModelInvisible(ControllerActionsEventArgs e)
             }
         }
 
+        public GameObject ControllerAlias {
```

Comment:
I added that method to the DeviceFinder and updated the properties on ControllerActions and ControllerEvents to utilize it.